srcpkgs/dwarffortress/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'dwarffortress'
+pkgname=dwarffortress
+version=0.44.12
```

Comment:
shouldn't it be 44.12 ?

Reply:
official versioning prepends 0.

Reply:
That is worrying since their URL doesn't do it.

Reply:
![dvers](https://user-images.githubusercontent.com/29682274/55970711-a09ba600-5cb2-11e9-9702-b8b230fb4e4c.png)
should it be ignored for convenience?

Reply:
Please keep their original versioning and make a temporary variable.

version=0.44.12
_urlver=${version#*.}
distfiles=${urlver//./_}

Reply:
```
version=0.44.12
urlver=${version#*.}
distfiles=${urlver//./_}
```
is working here